### ISSUE TITLE
Fix dev setup for Storybook & Stencil

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#3389]
Currently, the dev setup is broken. Storybook cannot be run in parallel with stencil. It would be really nice if this was fixed and we could work on the docs, the styles and the components in parallel again.

Ideas:
- Compile stencil components in the docs package (import source files, not from loader or dist)
- Ask for support from storybook/stencil/vite side on this

**Comment by gfellerph** [comment]
Closing as no longer relevant
